tests/flags_to_string_ut/flags_to_string_ut.c

```diff
@@ -107,6 +107,12 @@ TEST_FUNCTION_CLEANUP(method_cleanup)
     umock_c_negative_tests_deinit();
 }
 
+// For gcc build that causes a false negative
+#ifdef __GNUC__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstringop-overflow"
+#endif
```

Comment:
What is the error for this one?
Not sure we should suppress ....

Reply:
I believe this one will need to be suppressed.  I'll add information I get from the interwebs for the reasons.